lib/ios_platform_images.dart

```diff
@@ -8,22 +8,27 @@ import 'package:flutter/painting.dart';
 import 'package:flutter/foundation.dart'
     show SynchronousFuture, describeIdentity;
 
-// Performs exactly like a MemoryImage but instead of taking in bytes it takes
-// in a future that represents bytes.
+/// Performs exactly like a [MemoryImage] but instead of taking in bytes it takes
+/// in a future that represents bytes.
 class FutureMemoryImage extends ImageProvider<FutureMemoryImage> {
-  const FutureMemoryImage(this.futureBytes, {this.scale = 1.0})
-      : assert(futureBytes != null),
+  /// Constructor for FutureMemoryImage.  [_futureBytes] is the bytes that will
+  /// be loaded into an image and [scale] is the scale that will be applied to
+  /// that image to account for high-resolution images.
+  const FutureMemoryImage(this._futureBytes, {this.scale = 1.0})
+      : assert(_futureBytes != null),
         assert(scale != null);
 
-  final Future<Uint8List> futureBytes;
-
+  final Future<Uint8List> _futureBytes;
+  /// Scalar used to denote high resolution images.
```

Comment:
You didn't say what the numerical value means. I'm assuming this is pixel density.

Reply:
I don't really know how to describe it beyond that.  It's similar to Apple's docstring for the same concept: https://developer.apple.com/documentation/uikit/uiimage/1624110-scale?language=objc

Pixel density exactly correct because that would be measured in units like dpi.

Reply:
If the source of truth is vague, we can reference that source of truth. In other words, if I `FutureMemoryImage(..., {scale = x})`, how would I know what number to put in there since it's a public API?

Reply:
Copy and pasted docstring from MemoryImage.